data/leaderboard.py

```diff
@@ -53,7 +50,7 @@ def get_most_chess_games_won(self, req_item: GetMostChessGamesWonRequest, number
                 user_dict[user_str] = user
                 i += 1
                 req_item.set_response(number_of_games=number_of_games, data=user_dict)
-        except FailureException as e:
+        except DatabaseFailureException as e:
```

Comment:
Just making the FailureException a bit clearer about the fact that it pertains to database failures.